lib/rack/lint.rb

```diff
@@ -271,13 +271,28 @@ def check_env(env)
       ## accepted specifications and must not be used otherwise.
       ##
 
-      %w[REQUEST_METHOD SERVER_NAME SERVER_PORT
-         QUERY_STRING
+      %w[REQUEST_METHOD SERVER_NAME QUERY_STRING
          rack.version rack.input rack.errors
          rack.multithread rack.multiprocess rack.run_once].each { |header|
         assert("env missing required key #{header}") { env.include? header }
       }
 
+      ## The <tt>SERVER_PORT</tt> must be an integer if set.
```

Comment:
I found that some servers just set it to an empty string. There is a proviso it should be nil if empty somewhere in the spec. So... I don't know. I'd like to think if it's present, it's a valid integer. Otherwise, it's nil.

Reply:
I will implement the current behaviour in this PR, and created an issue to discuss further.